File: prob25/prob25.js
function time_difference(s,e) {
	return (e - s) / 1000 // sec
}

var start = new Date(2013, 1, 1,  16, 10);
var end = new Date(2013, 1, 1, 17, 10);

var dif = time_difference(start,end)


document.write(dif)

File: prob25/prob25.py
import datetime as dt

def time_difference(s,e):
    start_dt = dt.datetime.strptime(s, '%H:%M')
    end_dt = dt.datetime.strptime(e, '%H:%M')
    return end_dt - start_dt

start = "09:35"
end = "11:00"

print time_difference(start,end)

start = "10:45"
end = "11:00"

print time_difference(start,end)
 
